RFC-004-SWAP-Bitcoin.md

```diff
@@ -0,0 +1,78 @@
+# Bitcoin Ledger in SWAP Messages
+
+- RFC-Number: 004
+- Status: Draft
+- Discussion-issue: -
+- Created on: 1 Mar. 2019
+
+# Table of contents
+
+<!-- toc -->
+
+- [Description](#description)
+- [Motivation](#motivation)
+- [Content](#content)
+  - [Ledger definition](#ledger-definition)
+    - [Parameters](#parameters)
+  - [Asset definition](#asset-definition)
+- [Registry extension](#registry-extension)
+
+<!-- tocstop -->
+
+## Description
+
+This RFC introduces Ledger & Asset definitions to be used in SWAP Messages for [Bitcoin Core](https://github.com/bitcoin/bitcoin/) network.
+
+## Motivation
+
+This RFC aims to define the necessary values and parameters to support Bitcoin Core in SWAP messages.
+This, added to complimentary RFCs per SWAP protocol, allows the execution of SWAP involving assets defined on the Bitcoin Core Ledger.
+
+## Content
+
+### Ledger definition
+
+This RFC's objective is to uniquely identify the Ledger on which the SWAP will operate.
+It is a difficult task as soft and hard forks may happen and there is no guarantee that the conditions identifying today are valid tomorrow.
+
+We define Bitcoin Core as the network and consensus whose standard implementation is [bitcoind](https://github.com/bitcoin/bitcoin/).
+
+- Name: Bitcoin
+- Description: Bitcoin Core Network
```

Comment:
What does this `Description` refer to? I don't think we define it anywhere, do we?

Reply:
Not `description` but "description", the English word.

Reply:
What I mean is that it describes the item.

Reply:
> What I mean is that it describes the item.

I got that. I just don't understand why?

If, as you described above, want to refer to the `Ledger` type here without any further explanation, then the `Description` is misplaced IMO because a `Ledger` doesn't have a `Description`. It also seems redundant to because you said exactly that in the previous paragraph.

Reply:
note: I'm removing the `Description` in favour of just describing it in the actual section.